features/menu.feature

```diff
@@ -82,29 +81,25 @@ Feature: Top right menu
          -----END LISK SIGNED MESSAGE-----
          """
 
-  @ignore
   Scenario: should allow to exit sign message dialog
     Given I'm logged in as "any account"
     When I click "sign message" in main menu
     And I click "cancel button"
```

Comment:
When I remove this line, I expect the test to fail, but it passes. apparently this line doesn't affect the test result.
Same goes for the next 2 scenarios testing x and cancel buttons close our sign and verify message dialogs.